labs/dom/expand/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+   <head>
+      <meta charset="utf-8">
+      <title>Expand</title>
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+      <link rel="stylesheet" href="https://bootswatch.com/4/litera/bootstrap.min.css" type="text/css">
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.15.0/themes/prism-okaidia.min.css">
+      <link rel="stylesheet" href="/assets/css/basic.css"/>
+      <link rel="stylesheet" href="/assets/css/dom.css"/>
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
+   </head>
+   <body>
+      <main role="main" id="main">
+         <h1>Expand</h1>
+         <p class="lead"> Extend your collapse code to change the icon from <i class="material-icons">expand_less</i> to <i class="material-icons">expand_more</i> once the div is collapsed. The button should then expand the div when clicked on and revert the icon</p>
```

Comment:
Again, up to you whether you add the condition on a single event listener or not.